clients/java-logger/library/src/test/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessorTests.java

```diff
@@ -78,14 +92,31 @@ public void processShouldPopulateJustificationMetadata() throws Exception {
                         ListValue.newBuilder()
                             .addValues(Value.newBuilder().setStringValue("test-aud")))
                     .build())
+            .putFields(
+                "justs",
+                Value.newBuilder()
+                    .setListValue(
+                        ListValue.newBuilder()
+                            .addValues(Value.newBuilder().setStructValue(wantJustificationsClaim)))
+                    .build())
             .build();
     Struct wantMetadata =
         Struct.newBuilder()
             .putFields(
                 "justification", Value.newBuilder().setStructValue(wantJustification).build())
             .build();
+    RequestMetadata wantRequestMetadata =
+        RequestMetadata.newBuilder()
+            .setRequestAttributes(
+                Request.newBuilder()
+                    .setReason("[{\"category\":\"explanation\",\"value\":\"need-access\"}]")
```

Comment:
Does `[{"category":"explanation","value":"need-access"}]` work?

Reply:
No, I tried that, not sure why.

Reply:
What's the error message?

Reply:
```
java.lang.Error: Unresolved compilation problems: 
 The method setReason(String) in the type AttributeContext.Request.Builder is not applicable for the arguments (String, String)
 Syntax error on tokens, delete these tokens
```
looks like it still need escape for "" inside the sentence.